migrations/20170531132953_add-followers-to-communties_tags.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex, Promise) {
+  return knex.schema.table('communities_tags', table => {
+    table.integer('followers').defaultTo(0)
```

Comment:
i feel like `num_followers` would be a bit clearer, like we have `num_votes` for `posts`